### DISCUSSION TITLE
[Question]: Set a custom openai endpoint, it can't be used

### DISCUSSION THREAD

**Discussion by yuband** [danny-avila/LibreChat#3452] [Troubleshooting]
### What is your question?

How to use a custom OpenAI endpoint

### More Details

I customized an openai reverse proxy, I modified all the 'https://api.openai.com/v1/chat/completions' endpoints in the configuration to my own reverse proxy, after the container was started, it prompted that it could not be used, how should the customized endpoint be configured?

### Screenshots

<img width="868" alt="image" src="https://github.com/user-attachments/assets/0bd504fb-0c20-405b-99d0-c7d4e2de9e82">

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by bpawnzZ** [comment]
interested in this as well

**Comment by danny-avila** [comment]
make sure it's set correctly, i.e `OPENAI_REVERSE_PROXY=https://your-reverse.proxy` <--- the endpoint but not including "/chat" or "/completion" routes

reverse proxy value is set to `'https://api.openai.com/v1` as the "baseURL" (it doesn't _need_ the "v1" part, it's just the part of the URL that precedes `/chat/completions` because the app will append this)

**Reply by yuband** [reply]
It worked, thank you very much!
